manifests/config.pp

```diff
@@ -99,6 +102,22 @@
     fail('when rundeck::manage_home = false a file definition for the home directory must be included outside of this module.')
   }
 
+  if $manage_cli_config {
+    file{ "${rdeck_home}/.rd":
+      ensure  => directory,
+    }
+    file{ "${rdeck_home}/.rd/rd.conf":
+      ensure  => file,
+      content => @("END")
```

Comment:
Suggest using a template instead of there HEREDOC as templates can be validated.